contracts/DisintermediatedGrants.sol

```diff
@@ -115,21 +118,19 @@ contract DisintermediatedGrants is Ownable {
         }
     }
 
-    function proposeGrant(
-        uint256 _donationId,
-        address _recipient,
-        uint256 _amount
-    ) public onlyOwner {
-        Donation memory donation = donations[_donationId];
-        require(donation.amount - donation.disbursedAmount >= _amount, "donation cannot cover full grant amount");
+    function proposeGrant(GrantProposal memory _grantProposal) public onlyMultisig {
+        Donation memory donation = donations[_grantProposal.donationId];
```

Comment:
Add `require` that checks if donation with `donationId` exists

Reply:
Fixed in [github.com/researchdao/contracts/commit/377339512f875c3b18da257ac79e96f12fee3e76](https://github.com/researchdao/contracts/commit/377339512f875c3b18da257ac79e96f12fee3e76).